components/StoryCards/StoryCard.tsx

```diff
@@ -64,6 +64,7 @@ export function StoryCard({
                 title={titleAsString}
             >
                 <StoryImage
+                    aspectRatio={4 / 3}
```

Comment:
Ok, I can buy that :)

Do you think it's a good fix for the grid (non-masonry) layout?

Reply:
Yeah, for the grid layout should be okay as that's what we're using for the [image there](https://github.com/prezly/theme-nextjs-bea/blob/main/components/StoryCards/StoryCard.module.scss#L49).

Reply:
Made the change to only force image aspect ratio for the `grid` layout.

I've also tried using the **Uploadcare smart cropping functionality** to move the focus of the cropped image to a point of interest (instead of always centering). Here's the result:

# BEFORE


https://github.com/user-attachments/assets/0f5793f7-268f-4e53-94e6-c38c80e15230



# AFTER


https://github.com/user-attachments/assets/51ac7dad-9ccb-4e98-ad41-b62afbcacdd4

Reply:
Looks a bit better? But I can see the images are still blurry, at least it seems like that from the video, so not sure 🤔

Reply:
Yes, they are still blurry (but less blurry than they used to be). Most of them are 219px tall, while we display them in a 247px tall box.

Reply:
> Most of them are 219px tall, while we display them in a 247px tall box.

So it doesn't fix the issue, right? Also how does this smart crop work if there's actual crop on the image? I'm a bit worried about using the smart crop if people crop their images in a specific way. As it will affect almost every newsroom in Prezly.

I still think the simplest way to fix this is to request a larger width image instead of using the predefined value provided by [`getCardImageSizes`](https://github.com/prezly/theme-nextjs-bea/blob/main/components/StoryImage/lib.ts#L16).

So, if requesting an image with a width of 370px results in height of 219px, we need to request larger width to compensate. So, for example requesting a width of 440px that will result in height of 250px (just an example).

Reply:
Not saying that the smart crop is a bad option, but I would check with UX people and maybe also Christian/Dein, just to make them aware of this.

Reply:
> So it doesn't fix the issue, right?

Not fully, yes. But it makes it much better. Instead of scaling up a 69px tall image to 247px (~3.6x) we scale 219px tall image to 247px (1.12x). So it's much crispier.

Reply:
Humm, I like the improvements on crispier images, but I'm not sure of the smart crop. I think it's not nice to mess up with image cropping. I'd just put it center aligned (fill) as it's the normal behaviour when nothing is touched.

I do agree that the way forward should be to ask users for better images (maybe when uploading, but I understand that if they don't follow the instructions we can make them better for them. I would love to have a clearer answer, but I don't. I just don't feel confident of auto-cropping for the reason explained above.

Reply:
Ok, I'm reverting the smart cropping back to standard centered cropping.